lang/fr_FR.UTF-8/LC_MESSAGES/help.po

```diff
@@ -7231,20 +7231,3 @@ msgstr "URL complète permettant d'accéder à l'API du serveur central Centreon
 
 msgid "Allows to skip the SSL certificate check on the Centreon's central server."
 msgstr "Permet de ne pas vérifier le certificat SSL du serveur central Centreon."
-
-msgid ""
-"No: You don't consent to share your platform data. Contact Details: You consent to share your platform data including"
-"your alias and email. Anonymized: You consent to share your platform data, but your alias and email will be anonymized."
-msgstr ""
-"Non: Vous ne consentez pas à partager les informations de votre plateforme. Détails du Contact: Vous consentez"
-"à partager les informations de votre plateforme, y compris votre alias et email. Anonymisé: Vous consentez à partager"
-"les informations de votre plateforme, mais votre alias et votre email seront anonymisés."
-
-msgid ""
-"If checked this option will restore the use of the deprecated pages."
-"This includes display of the deprecated pages and internal redirections between pages"
-"If not checked this option will enable the full use of the new Monitoring page Resource Status"
-msgstr ""
-"Si l'option est activée l'utilisation des pages dépréciées sera totalement rétablie"
-"Cela inclu: l'affichage des pages mais également toutes les redirections entre pages dépréciées"
-"Si l'option n'est pas activée l'utilisation de la nouvelle page de Monitoring Resource Status sera pleinement activée"
```

Comment:
This part should not be deleted as it is part of something else (feature flipping)